src/commitments.rs

```diff
@@ -20,7 +21,7 @@ impl MultiCommitGens {
     let mut gens: Vec<GroupElement> = Vec::new();
     let mut uniform_bytes = [0u8; 64];
     for _ in 0..n + 1 {
-      reader.read(&mut uniform_bytes);
```

Comment:
(This probably wouldn't come up in practice with the shake code, but technically read could return an I/O error or not fill the entire buffer.)

Reply:
Got it, cool! 

Tests pass, but now there is a Cargo warning (I can fix this and make a commit to the PR). 

warning: unused import: `XofReader`
 --> src/commitments.rs:3:39
  |
3 | use digest::{ExtendableOutput, Input, XofReader};
  |                                       ^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default